frontend/src/app/plugins/utils.cljs

```diff
@@ -7,7 +7,14 @@
 (ns app.plugins.utils
   "RPC for plugins runtime."
   (:require
-   [app.util.object :as obj]))
+   [app.common.data.macros :as dm]
+   [app.common.uuid :as uuid]
+   [app.util.object :as obj]
+   [cuerdas.core :as str]
+   [promesa.core :as p]))
+
+(def uuid-regex
+  #"\w{8}-\w{4}-\w{4}-\w{4}-\w{12}")
```

Comment:
I think it would be better to use something like:

```clojure
#"[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
```

Reply:
done